DigitalLearningSolutions.Data.Migrations/202307040730_UpdateFirstnameLastNameUsersTable.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202307040730)]
+    public class UpdateFirstnameLastNameUsersTable : Migration
+    {
+        public override void Up()
+        {
+            Execute.Sql(@"Update Users set FirstName = TRIM(FirstName), LastName = TRIM(LastName)");
```

Comment:
TRIM was introduced in SQL 2017. The production SQL server is 2016. We will need to use `RTRIM(LTRIM(FirstName))` or similar to achieve this.

Reply:
Used the aforementioned as per compatibility with SSMS 2016